www/app/Services/Providers/AbstractCliProvider.php

```
@@ -0,0 +1,585 @@
+
+    /**
+     * Build the environment variables for the CLI process.
+     * Base implementation provides common env setup; subclasses extend.
+     *
+     * @param Conversation $conversation
+     * @param array $options
+     * @return array<string, string>
+     */
+    protected function buildEnvironment(
+        Conversation $conversation,
+        array $options
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$options'. (undefined)

(UnusedFormalParameter)